src/components/ui/CreateFormDropdown.tsx

```diff
@@ -10,6 +10,8 @@ import { useNavigate } from "react-router-dom";
 import { FormType } from "../../utils/types";
 import { menuItems } from "../../utils/data";
 
+type NextFormType = Exclude<FormType, "Application">;
```

Comment:
I would suggest renaming `NextFormType` to something more intuitive.
suggestions: 
1. `NonApplicationFormType`
2. `ApplicantTraineeFormType`